x-pack/test/performance/utils.ts

```diff
@@ -13,16 +13,13 @@ export function serializeApmGlobalLabels(obj: any) {
     .reduce((acc, [k, v]) => (acc ? `${acc},${k}=${v}` : `${k}=${v}`), '');
 }
 
-export async function waitForVisualizations(page: Page, visCount: number) {
-  await page.waitForFunction(function renderCompleted(cnt) {
+export function waitForVisualizations(page: Page, visCount: number) {
+  return page.waitForFunction(function renderCompleted(cnt) {
```

Comment:
Returning promises instead of using `async`/`await` has a minuscule performance benefit but hurts async stack traces and the ability to understand that future changes should turn this back to `async`/`await`. Highly recommend against this type of change.

Reply:
Oh, I see, this is just reverting to the previous state. 🙃